docs/_sources/vc_features.rst.txt

```diff
@@ -0,0 +1,36 @@
+VC features
```

Comment:
I wrote some solutions to windows related issues here if you want to use them, https://github.com/conda-forge/staged-recipes/wiki/Common-issues#python-27-x64-tests-on-windows-fails-when-using-cmakes-visual-studio-generator